cpp/ql/src/Security/CWE/CWE-611/XXE.ql

```diff
@@ -57,42 +57,51 @@ class XercesDOMParserClass extends Class {
 }
 
 /**
- * Gets a valid flow state for `XercesDOMParser` flow.
+ * The `SAXParser` class.
+ */
+class SAXParser extends Class {
```

Comment:
Any reason this is not called `SAXParserClass`? To be uniform with `XercesDOMParserClass` and `AbstractDOMParserClass`?

Reply:
No reason.  Would you prefer I rename this `SAXParserClass`, or rename `XercesDOMParserClass` to just `XercesDOMParser`?

Reply:
Just make it `SAXParserClass`.

Reply:
... I've gone with `SAXParserClass` for now (since it is a QL class _describing_ a C++ class) - but happy to change it.